plugin/evm/vm_test.go

```diff
@@ -2421,3 +2422,15 @@ func TestBlockGasValidation(t *testing.T) {
 		})
 	}
 }
+
+func TestInspectDatabases(t *testing.T) {
+	require := require.New(t)
+
+	var (
+		vm = newDefaultTestVM()
+		db = memdb.New()
+	)
+
+	vm.initializeDBs(db)
+	require.NoError(vm.inspectDatabases())
```

Comment:
(nit) since there is only 1 require usage 
```suggestion
	var (
		vm = newDefaultTestVM()
		db = memdb.New()
	)

	vm.initializeDBs(db)
	require.NoError(t, vm.inspectDatabases())
}
```